GDCpp/GDCpp/Extensions/Builtin/MathematicalTools.cpp

```diff
@@ -11,7 +11,7 @@ namespace GDpriv {
 namespace MathematicalTools {
 
 double GD_API normalize(double expression, double min, double max) { 
-  return (expression - min) / (max - min);
+  return (min = max ? max : (expression - min) / (max - min));

```

Comment:
min == max
Can you put an extra pair of parenthesis around the `(expression - min) / (max - min)`? I'm afraid otherwise it could be understand as `max/(max-min)` in the case `min == max`, it's always unclear what get the priority first.

Reply:
Done
I've made a test with an online sandbox for c++, but yeah it better to be sure.